plugins/modules/cloud/atomic/atomic_image.py

```diff
@@ -25,25 +25,28 @@
     backend:
         description:
           - Define the backend where the image is pulled.
-        choices: [ docker, ostree ]
+        choices: [ 'docker', 'ostree' ]
```

Comment:
Didn't we usually get rid of the quotes in YAML lists? (I'm not sure anymore... @Andersson007 what do you think?)

Reply:
@felixfontein sorry for delayed reply:( I just noticed it by chance.. For me personally it's always a good idea to avoid unnecessary things